ts/torch_handler/utils/conf/config.py

```diff
@@ -0,0 +1,63 @@
+from abc import abstractmethod

```

Comment:
Note to self: we have yaml parsing in quite a few parts of our codebase now from `benchmarks` to the new #1492 so may make sense to unify this at some point